packages/pushnotification/android/build.gradle

```
@@ -19,15 +19,19 @@ allprojects {
     }
 }
 
+def getExtOrIntegerDefault(name) {
+  return rootProject.ext.has(name) ? rootProject.ext.get(name) : 28
+}
+
```

Can we make this function a bit more generic to handle setting all of the version values?
```suggestion
def getExtOrDefault(name, default) {
  return rootProject.ext.has(name) ? rootProject.ext.get(name) : default
}
```

and then we can use it below for all of the versions:
```gradle
compileSdkVersion getExtOrDefault('compileSdkVersion', 28) 

buildToolsVersion getExtOrDefault('buildToolsVersion', "28.0.3") 

minSdkVersion getExtOrDefault('minSdkVersion', 16) 

targetSdkVersion getExtOrDefault('targetSdkVersion', 28) 
```

Also (as noted in the code above), we should bump up the `buildToolsVersion` as well.

---

Yeah great suggestion, that slipped me by. Will fix.